data/RedHat-9.yaml

```diff
@@ -0,0 +1,10 @@
+# Specific options for RedHat 8 systems
```

Comment:
s/8/9

Reply:
Good catch, this is fixed.

Reply:
Thank you! and thanks again for the updates.